server/templaterenderer/template.go

```diff
@@ -141,6 +141,9 @@ func (tr *templateRenderer) init() {
 {{- else}}[{{.NickName}}]({{.Links.HTML.Href}})
 {{- end -}}
 `))
+
+	//The user template links to the corresponding user in BitBucket.
```

Comment:
Nit: Add space for consistancy
```suggestion
	// The user template links to the corresponding user in BitBucket.
```